tests/foreman/api/test_contentmanagement.py

```diff
@@ -663,20 +625,18 @@ def test_positive_capsule_sync(
         # repository
         assert repo.content_counts['rpm'] == cvv.package_count
 
-        self.wait_for_sync(module_capsule_configured)
+        module_capsule_configured.wait_for_sync()
 
         # Assert that the content published on the capsule is exactly the
         # same as in repository on satellite
-        sat_repo_url = get_published_repo_url(
-            target_sat,
+        sat_repo_url = target_sat.get_published_repo_url(
```

Comment:
Does `target_sat` ever inherit the `CapsuleInfo` class or subclass ? I'm not sure about this.

Reply:
It does, yes.